docs/content/docs/test/FlakyTests.md

```diff
@@ -5,9 +5,23 @@ type: docs
 
 # Flaky Tests
 
-A flaky test is a test that could fail or pass for the same configuration.
-One could say this is unstable tests that fail sometimes but not every launch.
-Usually, it happens when a test relies on an external environment. Network or real back-end for instance.
+A flaky test is a test that could fail or pass without project changes.\
+One could say this is unstable tests that fail sometimes but not every launch.\
+It happens when: 
+1. test relies on an external environment. Real back-end for instance.
+1. test had written incorrect. Mistakes easy to do in async jobs.
+1. test framework has bugs
```

Comment:
https://avito-tech.github.io/avito-android/docs/contributing/docs/#%D0%BD%D1%83%D0%BC%D0%B5%D1%80%D0%BE%D0%B2%D0%B0%D0%BD%D0%BD%D1%8B%D0%B9-%D0%B8%D0%BB%D0%B8-%D0%BD%D0%B5%D0%BD%D1%83%D0%BC%D0%B5%D1%80%D0%BE%D0%B2%D0%B0%D0%BD%D0%BD%D1%8B%D0%B9